pkg/plugins/storage/fs_test.go

```diff
@@ -48,89 +48,6 @@ func TestAdd(t *testing.T) {
 	require.Equal(t, files[5].Name(), "text.txt")
 }
 
-func TestRemove(t *testing.T) {
```

Comment:
Do we need to re-implement this test with `LocalFS`?

Reply:
Thanks for keeping me honest 😇